### ISSUE TITLE
Question 12

### ISSUE THREAD

**Issue by emiesen** [guy915/Computational-Intelligence-Models#14]
Pick the architecture with the best results and plot its performance on the training set and the validation set during training, across epochs. Justify your choice.

Rubric 12:
• (2 pt.) Correct and readable plot.
• (1 pt.) Correct choice of architecture based on previous results.
• (1 pt.) Sensible justification of this choice.
